src/Config/ConfigPanel.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
As commented before we may need to add copyright from the source this code was referenced.
@mhs4670go , please check.

Reply:
https://github.com/microsoft/vscode-extension-samples
In this sample, they says "Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License."
this license will be problem?

Reply:
1. 기존 오픈소스를 사용하는 경우, 파일의 소스코드 상단에 명시되어 있는 저작권 표시문구 (프로그램 이름, 작성 일자, 개발자 이름, 버전, 연락처 등)와 라이선스 정보(라이선스 명, 라이선스 버전 등)를 임의로 삭제하거나 수정하지 않아야 합니다.
2. 여러 오픈소스 라이선스를 사용한 경우, 각 파일 상단에는 기존 저작권 표시문구와 라이선스 정보를 유지하고, 최상위 디렉토리 내 COPYING, LICENSE파일 등에 사용한 모든 라이선스 전문을 포함합니다. (단, 이 경우 각각의 소스파일 내에는 “해당 파일에 대한 라이선스 정보를 확인하려면 최상위 디렉토리 내 COPYING, LICENSE 파일을 참조하라.”는 간략한 문구를 삽입해도 됩니다.)

This is the guide of license. Since there is no license contents in original file, you can go as it is. And, I'll post a PR that adds MIT copyright to root LICENSE file.

Reply:
thank you so much then, i will change only nonce part